ppr-ui/src/components/common/PartyReview.vue

```diff
@@ -0,0 +1,142 @@
+      </header>
+    </slot>
+
+    <div :class="{ 'border-error-left': showIncomplete }">
+      <!-- Incomplete Section Msg -->
+      <section v-if="showIncomplete" class="mx-6 py-9">
+        <v-icon color="error">mdi-information-outline</v-icon>
+        <span class="error-text mx-1">This step is unfinished.</span>
+        <router-link :to="{ path: returnRoute }">Return to this step to complete it.</router-link>
+      </section>
+
+      <!-- Party Info -->
+      <template v-if="hasData">
+
+        <section class="party-info fs-14">
+          <!-- Upper party info slot -->
+          <slot name="topInfoSlot" />
+
+          <!--- Party Info Headers -->
+          <v-divider class="mx-4"/>
+          <v-row no-gutters class="px-6 pt-6 pb-2">
+            <!-- Future: Handle person name -->
+            <v-col v-if="hasPropData('businessName')">
+              <h3>Name</h3>
+            </v-col>
+            <v-col v-if="hasPropData('address')">
+              <h3>Mailing Address</h3>
+            </v-col>
+            <v-col v-if="hasPropData('emailAddress')">
+              <h3>Email Address</h3>
+            </v-col>
+            <v-col v-if="hasPropData('phoneNumber')">
+              <h3>Phone Number</h3 >
+            </v-col>
+          </v-row>
+          <v-divider class="mx-4"/>
+
+          <!-- Party Info Data -->
+          <v-row no-gutters class="px-6 py-7">
+            <v-col v-if="hasPropData('businessName')">
+              <!-- Future: Handle person name -->
+              <label class="generic-label fs-14">
+                <v-icon class="mt-n2 mr-1">mdi-domain</v-icon>
+                {{ partyModel.businessName || '(Not Entered)' }}
+              </label>
+            </v-col>
+
+            <v-col v-if="hasPropData('address')">
+              <base-address
+                v-if="hasTruthyValue(partyModel.address)"
+                :value="partyModel.address"
+              />
+
+              <p v-else> (Not Entered) </p>
+            </v-col>
+
+            <v-col v-if="hasPropData('emailAddress')">
+              <p>{{ partyModel.emailAddress || '(Not Entered)'}}</p>
+            </v-col>
+
+            <v-col v-if="hasPropData('phoneNumber')">
+              <p>
+                {{ partyModel.phoneNumber || '(Not Entered)'}}
+                {{ partyModel.phoneExtension ? `Ext ${partyModel.phoneExtension}` : '' }}
+              </p>
+            </v-col>
+          </v-row>
+
+          <!-- Bottom party info slot -->
+          <slot name="bottomInfoSlot" />
+        </section>
+      </template>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, reactive, toRefs } from 'vue-demi'
+import { RouteNames } from '@/enums'
+import { BaseAddress } from '@/composables/address'
+import { PartyIF } from '@/interfaces'
+import { hasTruthyValue } from '@/utils'
+
+export default defineComponent({
+  name: 'PartyReview',
```

Comment:
This can replace our wide variety of Party review components, which are all so similar but tightly coupled to whatever flow they are implemented. 

As you can see there is no local styling, it's entirely based on our `base.scss` and vuetify helper classes. 
Meaning that we can use all over and start to create some consistency. 

This component will grow to include some of the additional `Party` properties but should leverage the slots or create new ones if required for the flow specific items. See the example below for QS Information.